package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@trayio/falafel",
-  "version": "1.24.3",
+  "version": "1.25.0-alpha.0",
```

Comment:
Want to release this initially as beta, but need to test if this works with connector builder/deployer.